packages/patternfly-4/react-core/src/components/Title/__tests__/Title.test.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import { Title, TitleSize, TitleLevel } from '.';
+import { Title, TitleSize, TitleLevel } from '..';
```

Comment:
../Title ?

Reply:
It implicitly grabs the exports from the barrel file index.tsx